### DISCUSSION TITLE
How to import the project as a whole in VSCode

### DISCUSSION THREAD

**Discussion by pinkfrog9** [uxbox/uxbox#1285] [Q&A]
Hi,
I am developing in VSCode with Calva+Clojure-LSP.

I can import the following three projects separately, namely:
1. backend
2. frontend
3. common

How can I import the project as a whole instead of creating three projects, much like in intellij where there is the concept of modules, and the module here could be backend/fronted/common.

**Comment by ajsb85** [comment]
### Multi-root Workspaces
_You can work with multiple project folders in Visual Studio Code with multi-root workspaces._
https://code.visualstudio.com/docs/editor/multi-root-workspaces
